larg4/Core/larg4Main_module.cc

```diff
@@ -124,8 +126,9 @@ larg4::larg4Main::larg4Main(fhicl::ParameterSet const & p)
   session_(0),
   UI_(0),
   seed_(p.get<long>("seed", -1)),
```

Comment:
Same comment as immediately below

Reply:
There are no changes here WRT current functionality.  If removing the default is desired, then that should be proposed at a LArSoft coordination meeting.

Reply:
Oops, this one was my bad.  I swear it was highlighted in green earlier...